# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cxxtools/hotfix-time.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cxxtools-3.0/src/timer.cpp.vanilla	2025-06-24 21:52:36.851960208 +0200
+++ cxxtools-3.0/src/timer.cpp	2025-06-24 21:53:20.575704344 +0200
@@ -28,6 +28,7 @@
 #include "cxxtools/selector.h"
 #include "cxxtools/datetime.h"
 #include <stdexcept>
+#include <ctime>
 
 namespace cxxtools
 {
